src/tests/reducers/likeReducer.test.js

```diff
@@ -0,0 +1,146 @@
+import * as types from '../../redux/constants/articles';
+import likeReducer, { updateCount } from '../../redux/reducer/article/likes';
+
+const initialState = {
+  articleLiked: false,
+  articleDisliked: false,
+};
+
+const reaction = {
+  status: 'like',
+  likeCount: 0,
+  dislikeCount: 0
+};
+
+describe.only('Like Reducer', () => {
```

Comment:
Nice job on increasing your test coverage. Please why do you still have `only` on the describe block.

Reply:
@blackshady The `only` has been removed. It was an oversight.

Reply:
Okay great!